### PR TITLE
Bug 1986504: bump openshift/api

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @stlaz

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,23 +9,23 @@ require (
...
+	github.com/openshift/build-machinery-go v0.0.0-20210712174854-1bb7fd1518d3
 	github.com/openshift/client-go v0.0.0-20210409155308-a8e62c60e930
 	github.com/openshift/library-go v0.0.0-20210414082648-6e767630a0dc
 	github.com/pkg/profile v1.4.0 // indirect
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
-	k8s.io/api v0.21.0
+	k8s.io/api v0.22.1
 	k8s.io/apiextensions-apiserver v0.21.0
```
Comment (p0lyn0mial) [inline go.mod:22]: shouldn't all deps be on the same level `0.22.1` ?

Comment (s-urbaniak) [reply go.mod]: fixed

**Pull request merged**